src/main/java/com/cb/controllers/ShopController.java

```diff
@@ -0,0 +1,95 @@
+package com.cb.controllers;
+
+import com.cb.bl.UserBL;
+import com.cb.dto.DefaultDTO;
+import com.cb.services.dbService.dbServiceImpl.ShopDBServiceImpl;
+import com.cb.services.dbService.dbServiceImpl.UserDBServiceImpl;
+import com.cb.services.dbService.iDbService.FighterDBService;
+import com.cb.services.dbService.iDbService.ShopDBService;
+import com.cb.services.mapService.iMapService.ShopService;
+import com.cb.services.mapService.iMapService.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@Controller
+public class ShopController {
+
+    @Autowired
+    UserDBServiceImpl userDAL;
```

Comment:
Why do you declare variable userDAL by UserDBServiceImpl? Wouldn't be better if you just do like this - "UserDAL userDAL"?

Reply:
Apparently this is not needed at all for ShopController. Deleted it